components/header_footer_route/content_layouts/alternate_link.tsx

```diff
@@ -9,12 +9,13 @@ import './alternate_link.scss';
 
 type AlternateLinkProps = {
     className?: string;
+    onClick?: any;
```

Comment:
it is a good practice to avoid as much as possible the use of `any`, you can define the type of the function like

`onClick?: () => void `(or if it will return an object, the object type)